charm_internal_version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
@phvalguima - this is the internal revision I mentioned yesterday.

**Problem**
In local deployments we are unable to tell if the the shards/config-server are the same revision / source code. (A requirement of sharded deplyoments)

This is an additional problem when we upgrade a cluster. Without knowing if their revision / source code is the same we do not know at what point a cluster has all the same source code. 

**Solution**
Add an internal revision number so that we can know the version of source code someone is using (only used in local deployments) 

**TL;DR**
The purpose of this is to have a better idea of what version a local charm is. (it does not need to match what is on charm hub)